src/calculateRentalCost.js

```diff
@@ -1,10 +1,23 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
+const SHORT_TERM = 3;
+const LONG_TERM = 7;
+const RENT_PER_DAY = 40;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM_DISCOUNT = 20;
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentPeriod = days;
+
+  if (rentPeriod >= LONG_TERM) {
+    return RENT_PER_DAY * rentPeriod - LONG_TERM_DISCOUNT;
+  }
+
+  if (rentPeriod >= SHORT_TERM) {
+    return RENT_PER_DAY * rentPeriod - SHORT_TERM_DISCOUNT;
+  }
+
+  if (rentPeriod < SHORT_TERM) {
+    return RENT_PER_DAY * rentPeriod;
+  }
```

Comment:
Function should always have return
What happened if all ifs is false?

<img width="1072" alt="Screenshot 2025-04-24 at 14 49 00" src="https://github.com/user-attachments/assets/7b8edcaf-e321-4555-9931-8add3fc959b8" />


```suggestion
    return RENT_PER_DAY * rentPeriod;
```